gitster.zsh-theme

```diff
@@ -13,13 +13,13 @@ typeset -gA git_info
 if (( ${+functions[git-info]} )); then
   zstyle ':zim:git-info:branch' format '%b'
   zstyle ':zim:git-info:commit' format '%c'
-  zstyle ':zim:git-info:clean' format '%F{green}✓'
-  zstyle ':zim:git-info:dirty' format '%F{yellow}✗'
+  zstyle ':zim:git-info:clean' format '%F{green}%{%G✓%}'
+  zstyle ':zim:git-info:dirty' format '%F{yellow}%{%G✗%}'
   zstyle ':zim:git-info:keys' format \
       'prompt' ' %F{cyan}%b%c %C%D'
 
   autoload -Uz add-zsh-hook && add-zsh-hook precmd git-info
 fi
 
-PS1='%(?:%F{green}:%F{red})➜ %F{white}$(prompt-pwd)${(e)git_info[prompt]}%f '
+PS1='%(?:%F{green}:%F{red})%{%G➜%}  %F{white}$(prompt-pwd)${(e)git_info[prompt]}%f '
```

Comment:
```suggestion
PS1='%B%(?:%F{green}:%F{red})%{%G➜%} %F{white}$(prompt-pwd)${(e)git_info[prompt]}%f%b '
```
Hi, thanks for the fix and for checking our prompt against the original. I agree with adding the bold to match it. I don't like the extra space character. I see that's also in the original, but I'm not sure it's intentional.  :- )

Reply:
Let's do it ! I will just make a local patch with the extra space then 😉